### PR TITLE
Entri - Nostra template created

### PR DESCRIPTION
**Pull request by leopinzon**

### CODE REVIEW

File: goentri.com.nostra.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "serviceId": "nostra",
+  "serviceName": "Nostra",
+  "version": 1,
+  "logoUrl": "https://cdn.goentri.com/logo.svg",
+  "description": "Allows user to easily set up domain using Entri",
+  "syncPubKeyDomain": "goentri.com",
+  "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com,app.goentri.com",
+  "variableDescription": "mappingValue is the nostra's target domain",
+  "sharedProviderName": true,
+  "hostRequired": false,
```
Comment (pawel-kow) [inline goentri.com.nostra.json:13]: hostRequired shall be true if CNAME is at APEX

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: goentri.com.nostra.json (hunk 1 above)
Comment (leopinzon) [reply goentri.com.nostra.json]: Perfect @pawel-kow , set to true, thanks!

Comment (leopinzon) [reply goentri.com.nostra.json]: let me know if there something left to fix here @pawel-kow . thanks a lot

**Review by pawel-kow — APPROVED** [review]
Thx

**Pull request merged**